core/trino-main/src/main/java/io/trino/testing/LocalQueryRunner.java

```diff
@@ -31,11 +31,13 @@
 import io.trino.connector.system.GlobalSystemConnector;
 import io.trino.connector.system.GlobalSystemConnectorFactory;
 import io.trino.connector.system.MaterializedViewPropertiesSystemTable;
+import io.trino.connector.system.MaterializedViewSystemTable;
 import io.trino.connector.system.NodeSystemTable;
 import io.trino.connector.system.SchemaPropertiesSystemTable;
 import io.trino.connector.system.TableCommentSystemTable;
 import io.trino.connector.system.TablePropertiesSystemTable;
 import io.trino.connector.system.TransactionsSystemTable;
+import io.trino.connector.system.TrinoMaterializedViewAdapter;
```

Comment:
I don't see this class in the PR

Reply:
The class is `TrinoMaterializedViewsSystemTableAdapter`. I'll update the PR ordering.